### PR TITLE
feat: dx updates to default demo site

### PR DESCRIPTION
**Pull request by orinokai**
### Summary

While testing the various rendering modes I made a few changes to the demo pages so that they were easier to test and it was more obvious how each page should be rendered. I thought it might be good to push these changes, if everyone agrees they are useful?

* Added descriptive text at the top of each page with a dynamic path that explains which paths are prerendered and what fallback options are enabled for other slugs
* Made the prerendered paths more consistent (always IDs 1 and 2)
* Added some package scripts for testing with `next build && next start`
* Added a simple VS Code launch.json for attaching the debugging to running inspector process

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

chill cat

### CODE REVIEW

**Review by ascorbic — DISMISSED** [review]
Much more coherent!

File: demos/default/package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "private": true,
   "description": "",
   "scripts": {
-    "build": "next build"
+    "build": "next build",
+    "dev": "next dev",
+    "serve": "next build && next start"
```
Comment (ericapisani) [inline demos/default/package.json:9]: [dust] would be great if we had a 'clean' command as well that removes all of the build artifacts as there's been times where some weird problems have resolved themselves after having those build artifacts removed prior to a build

**Review by ericapisani — APPROVED** [review]
These are great changes, thanks for doing this!

File: demos/default/package.json (hunk 1 above)
Comment (orinokai) [reply demos/default/package.json:9]: good point, though afaik there is no Next command for that? i have added a draft issue for it

**Review by MarcL — APPROVED** [review]
LGTM! 🎉

**Pull request merged**